lightning/tidb-lightning.toml

```diff
@@ -97,17 +97,19 @@ driver = "file"
 # - "replace": When encountering conflicting primary or unique key records, TiDB Lightning retains the latest data and overwrites the old data.
 #              The conflicting data are recorded in the `lightning_task_info.conflict_error_v2` table (recording conflicting data detected by post-import conflict detection in the physical import mode)
 #              and the `conflict_records` table (recording conflicting data detected by preprocess conflict detection in both logical and physical import modes) of the target TiDB cluster.
+#              If you turn on both preprocess and post-import conflict detection in physical import mode, the conflicting data can be checked in `lightning_task_info.conflict_view` view.
 #              You can manually insert the correct records into the target table based on your application requirements. Note that the target TiKV must be v5.2.0 or later versions.
 # - "ignore": When encountering conflicting primary or unique key records, TiDB Lightning retains the old data and ignores the new data. This option can only be used in the logical import mode.
 strategy = ""
 # Controls whether to enable preprocess conflict detection, which check conflicts in the data before importing it to TiDB. In scenarios where the ratio of conflict records is greater than or equal to 1%, it is recommended to enable preprocess conflict detection for better performance in conflict detection.
 # In other scenarios, it is recommended to disable it. The default value is false, indicating that TiDB Lightning only checks conflicts after the import. If you set it to true, TiDB Lightning checks conflicts both before and after the import. This parameter is experimental, and it can be used only in the physical import mode.
 # precheck-conflict-before-import = false
-# Controls the maximum number of conflict errors that can be handled when strategy is "replace" or "ignore". You can set it only when strategy is "replace" or "ignore". The default value is 9223372036854775807, which means that almost all errors are tolerant.
-# threshold = 9223372036854775807
```

Comment:
I remember the reason of separate `threshold` and `max-record-rows` is the user wants the import task success for the first time, but the user did not know how many duplicate records in it. So `threshold` should be set to a large value.

But if we record too many rows in the conflict table, the performance is not affordable, so there's another configuration `max-record-rows`.

How to resolve this case now?

Reply:
This change was determined by PM. The default `threshold`/`max-record-rows` is now set to 10000. In most scenarios, if the user wants to import data with more than 10000 duplicate records, there is probably something wrong with the data source, and the previous experience is that maybe the configuration is not set correctly, or the downstream table structure pk or uk is not well defined. So there is no meaning to keep importing although not further recording conflict data.
The change is the first step that we are gradually deprecating `max-record-rows`, which is to directly set the value of `max-record-rows` to be that of `threshold`.